bosch_locator_bridge_utils/params/params.yaml

```
@@ -0,0 +1,25 @@
+locator_node:
+  ros__parameters:
+    use_sim_time: False
+    base_frame_id: "base_footprint"
```

```suggestion
    base_frame_id: "base_link"
```
I think base_link would be a more generic default value.

---

Actually the 2D localization computes the pose of **base_footprint** relative to the map. When setting base_link as the base frame, the robot would sink a bit into the ground in RViz, if the z-position of base_link is greater than zero.